tests/Unit/IO/H5/Test_VolumeData.cpp

```diff
@@ -99,6 +98,37 @@ void test_strahlkorper() {
        "InertialCoordinates_z", "TestScalar"},
       {{0, 1, 2, 3}}, {}, observation_values[0]);
 
+  // Check pole connectivity
+  DataVector connectivity_data{};
+  {
+    h5::H5File<h5::AccessType::ReadOnly> strahlkorper_file{h5_file_name};
+    const auto& volume_file =
+        strahlkorper_file.get<h5::VolumeData>("/element_data", version_number);
+    const auto h5_connectivity =
+        volume_file.get_tensor_component(4444, "pole_connectivity").data;
+    connectivity_data = get<0>(h5_connectivity);
+    strahlkorper_file.close_current_object();
+  }
+
+  // Every 3 numbers is a triangle:
+  // root, root + n * (l + 1), root * (n + 1) * (l + 1)
+  std::vector<size_t> expected_connectivity = {0, 5, 10, 0, 10, 15, 0, 15, 20,
+                                               4, 9, 14, 4, 14, 19, 4, 19, 24};
+
+  // Store file connectivity in vector like expected_connectivity
+  std::vector<size_t> file_connectivity(expected_connectivity.size(), 0);
+
+  for (size_t i = 0; i < expected_connectivity.size(); i++) {
+    file_connectivity[i] = static_cast<size_t>(connectivity_data[i]);
+  }
```

Comment:
Define the expected value with the correct type so you don't have to do these conversions.